content/productivity/custom-distribute-silver/plugin.js

```diff
@@ -43,9 +47,9 @@ var removeAllChildNodes = (parent) => {
 
 // makes a (sometimes) clickable planet link
 var planetLink = (locationId, clickable = true) => {
-    const planet = df.getPlanetWithId(locationId);
+    const planet = df.entityStore.getPlanetWithId(locationId);
```

Comment:
Don't reach into entityStore. This function still exists and actually safeguards the ID isn't undefined.

Reply:
Done ✔️ 
Will stay clear from entityStore from now on 👌🏽